### ISSUE TITLE
Docs accessibility checker failure - contrast ratio

### ISSUE THREAD

**Issue by micahellison** [jrnl-org/jrnl#1934]
### Diagnostic output

n/a

### Current Behavior

It's failing on formats.md, though the problem is with colors so it's likely in the style:

```
 http://127.0.0.1:8000/formats/:

 • This element has insufficient contrast at this conformance level. Expected a
   contrast ratio of at least 4.5:1, but text in this element has a contrast
   ratio of 1.25:1. Recommendation:  change text colour to #fdfeff.

   (html > body > div:nth-child(1) > section > div > div > div:nth-child(2) >
   div > pre:nth-child(54) > code > span:nth-child(1) > span)

   <span class="hljs-string">"1.0"</span>
```

### Expected Behavior

n/a

### Repro Steps

n/a

### Debug output

n/a
